src/main/java/org/broadinstitute/hellbender/tools/walkers/validation/CountFalsePositives.java

```diff
@@ -20,8 +20,8 @@
 import java.util.List;
 
 @CommandLineProgramProperties(
-        summary = "Count PASS (false positive) variants in a vcf file for Mutect2 NA12878 normal-normal evaluation",
-        oneLineSummary = "Count PASS variants",
+        summary = CountFalsePositives.USAGE_SUMMARY,
+        oneLineSummary = CountFalsePositives.USAGE_ONE_LINE_SUMMARY,
         programGroup = VariantProgramGroup.class
```

Comment:
The program group should be VariantEvaluationProgramGroup. The doc portion including the example command isn't showing up in the javaDoc. Rather it looks like:

![screenshot 2018-01-03 21 57 42](https://user-images.githubusercontent.com/11543866/34548640-2a0ada40-f0d1-11e7-85ea-7e0997ba7670.png)

Reply:
Fixed